### ISSUE TITLE
First Sunday each month

### ISSUE THREAD

**Issue by ecrmnn** [mtdowling/cron-expression#85]
Hi

I'm having trouble getting next run dates for "first Sunday of each month".

Minutes, Hours, Day of month, Month, Day of Week
`0 12 ? * MON#1` - First Monday each month - Works 
`0 12 ? * 2#1` - First Tuesday each month - Works 
`0 12 ? * 5#1` - First Friday each month - Works 
`0 12 ? * 0#1` - First Sunday each month - Don't work 
`0 12 ? * SUN#1` - First Sunday each month - Don't work 

I'm getting this error:
```
InvalidArgumentException
Weekday must be a value between 1 and 5. 0 given
```

What expression do I need to get the first Sunday of each month?

**Comment by funivan** [comment]
I try this code and all is ok:
```php
$cron = \Cron\CronExpression::factory('0 12 ? * SUN#1');
```
Update to latest stable version. Mine is  v1.0.4

**Comment by charnad** [comment]
This has been fixed in #20

**Comment by holtkamp** [comment]
So @ecrmnn, this can be closed?
